lang/php/Portfile

```diff
@@ -185,11 +185,11 @@ switch ${subport_branch} {
     }
     8.3 {
         epoch           2
-        version         8.3.12
+        version         8.3.27
         use_xz          yes
-        checksums       rmd160  fd7d34f0528bcebd1ab33ff180bab40c4f2cd466 \
-                        sha256  f774e28633e26fc8c5197f4dae58ec9e3ff87d1b4311cbc61ab05a7ad24bd131 \
-                        size    12493432
+        checksums       rmd160  e79de6cce24967577d426c8756a4c35eb9016187 \
+                        sha256  c15a09a9d199437144ecfef7d712ec4ca5c6820cf34acc24cc8489dd0cee41ba \
```

Comment:
If it helps, I confirmed that this hash matches the php-8.3.27.tar.xz entry here: https://www.php.net/downloads.php?usage=web&os=osx&osvariant=osx-macports&version=8.3&source=Y